src/main/kotlin/uk/gov/dluhc/printapi/messaging/service/CertificateFailedToPrintEmailSender.kt

```diff
@@ -0,0 +1,41 @@
+package uk.gov.dluhc.printapi.messaging.service
+
+import mu.KotlinLogging
+import org.springframework.stereotype.Component
+import uk.gov.dluhc.printapi.client.ElectoralRegistrationOfficeManagementApiClient
+import uk.gov.dluhc.printapi.database.entity.Certificate
+import uk.gov.dluhc.printapi.dto.SendCertificateFailedToPrintEmailRequest
+import uk.gov.dluhc.printapi.messaging.models.ProcessPrintResponseMessage
+
+private val logger = KotlinLogging.logger {}
+
+@Component
```

Comment:
This is within service package, so should this be `@Service` instead of a `@Component`?
Also Service be appended to class name?


- Also applies to `CertificateNotDeliveredEmailSender` class below

Reply:
I'm on the fence slightly here. In terms of the annotations, it's essentially just syntactic sugar and you could argue this isn't a traditional "service" type class which interacts with a repository and executes transactions etc.